doc/configcommands.dsv

```diff
@@ -10,7 +10,7 @@ bookmark-cmd||<command>||""||If set, then <command> will be used as bookmarking
 bookmark-interactive||[yes/no]||no||If set to `yes`, then the configured bookmark command is an interactive program.||bookmark-interactive yes
 browser||<command>||%BROWSER, otherwise lynx||Set the browser command to use when opening an article in the browser. If the <<BROWSER,`BROWSER`>> environment variable is set, it will be used as the default browser, otherwise lynx will be used. For more information, see <<_using_browser,Using Browser>>.||browser "w3m %u"
 cache-file||<path>||"~/.newsboat/cache.db" or "~/.local/share/cache.db" (see "Files" section)||This configuration option sets the cache file. This is especially useful if the filesystem of your home directory doesn't support proper locking (e.g. NFS).||cache-file "/tmp/testcache.db"
-cleanup-on-quit||[yes/no]||yes||If set to `yes`, then the cache gets locked and superfluous feeds and items are removed, such as feeds that can't be found in the urls configuration file anymore.||cleanup-on-quit no
+cleanup-on-quit||[yes/no]||yes||If set to `yes`, then the cache gets locked and superfluous feeds and items are removed, such as feeds that can't be found in the urls configuration file anymore. Run `newsboat --cleanup` to do this manually. If you encounter a warning about unreachable feeds having been found, you may see the feed urls listed by creating a log file.||cleanup-on-quit no
```

Comment:
> by creating a log file

There are two log files in Newsboat: the one created with `--log-file` on the command line, and the one created with the `error-log` setting. The former is more of a debug facility, while the latter is user-facing; I'd prefer to keep it that way. So how about writing this info into the error log instead? For that, you have to log at `Level::USERERROR`, and adjust the text above to mention `error-log`.

Reply:
Using `USERERROR`-level and mentioning the `error-log` option now.